brooklyn-tosca-transformer/src/main/java/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaNodeToEntityConverter.java

```diff
@@ -388,17 +427,16 @@ protected void applyLifecycle(Map<String, Operation> ops, String opKey, EntitySp
                 try {
                     Path csarPath = csarFileRepository.getCSAR(artifact.getArchiveName(), artifact.getArchiveVersion());
                     script = new ResourceUtils(this).getResourceAsString(csarPath.getParent().toString() + "/expanded/" + ref);
-                } catch (CSARVersionNotFoundException e) {
+                } catch (CSARVersionNotFoundException | NullPointerException  e) {
                     script = new ResourceUtils(this).getResourceAsString(ref);
```

Comment:
this is the bit which should allow us to get external `https` items as well as search the classpath?